docs/drivers.rst

```diff
@@ -378,6 +378,15 @@ These sensors measure the ``distance`` to another object and may also measure li
     VL6180x 5 - 100 mm <https://circuitpython.readthedocs.io/projects/vl6180x/en/latest/>
     VL53L0x ~30 - 1000 mm <https://circuitpython.readthedocs.io/projects/vl53l0x/en/latest/>
 
+Ethernet
```

Comment:
I don't think Ethernet needs its own section at this point. How about grouping Wiznet, LoRa, ESP32SPI, Requests, WSGI (if it's there), and maybe other stuff into a Networking/Connectivity section?

Reply:
LoRa modules (rfm9x, rfm69) and esp32 (spi/at-command variant) are grouped into the Radio section below, on L390. Ethernet does not fit this categorization. WSGI is not a driver and I don't think it should be grouped into this.

Reply:
Ok, well my point is Ethernet is overly specific and shouldn't be in it's own section. Please find somewhere else to put it

Reply:
I don't think any of the current Driver sections fit its utility. I could put it in miscellaneous until we have a need for a dedicated Ethernet section.

Reply:
That's fine, I guess? I thought it would be helpful to reorganize the sections a bit but I can do that later.

Reply:
It would be helpful - could we re-organize in a separate PR? I have some ideas about organizing the drivers related to connectivity w.r.t. how specific we want to get.

Reply:
I reorganized the `helpers` section somewhat in #228, and created an `Internet/IoT` section. The categories in `helpers` might be helpful.

Reply:
Thanks @dhalbert. I'll merge this and then we can talk about reorganization in another ticket